libs/blocks/modal/modal.css

```diff
@@ -315,6 +338,37 @@
     overflow: hidden;
     width: 80%;
   }
+
+  .dialog-modal.tall-video {
+    max-width: 35vw;
+  }
+
+  .dialog-modal.tall-video .milo-video {
+    min-width: 35vw;
+  }
+}
+
+@media (min-width: 1400px) {
+  .dialog-modal.tall-video {
+    max-width: 25vw;
+    max-height: 1200px;
+  }
+
+  .dialog-modal.tall-video .milo-video {
+    min-width: 25vw;
+    max-height: 1200px;
+    height: 100%;
+  }
+}
+
+@media (min-width: 2000px) {
+  .dialog-modal.tall-video {
+    max-width: 22vw;
+  }
+
+  .dialog-modal.tall-video .milo-video {
+    min-width: 22vw;
+  }
```

Comment:
These feel a bit repetitive throughout the file. How about we just declare:
```
  .dialog-modal.tall-video {
    max-width: var(--modal-width-variable);
  }

  .dialog-modal.tall-video .milo-video {
    min-width: var(--modal-width-variable);
  }
```
and declare the variables themselves to adapt based on viewport?

Reply:
Interesting. I'll give that a try.

Reply:
Updated.